packages/retail-ui/components/CurrencyInput/CurrencyInput.tsx

```diff
@@ -18,6 +18,7 @@ export type CurrencyInputProps = Override<
     value: Nullable<number>;
     /** Кол-во цифр после зяпятой */
     fractionDigits?: Nullable<number>;
+    trailingZeros?: boolean;
```

Comment:
```suggestion
    /** Сохранять ли лишние нули после запятой */
    trailingZeros?: boolean;
```

Комментарий для доки не помешает.

Reply:
Добавил